application-admintools-ui/src/main/resources/AdminTools/Code/Macros.xml

```diff
@@ -152,5 +152,17 @@
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+#macro (tableOfContentsUI)
+  &lt;div class="adminToolsDashboardHeader"&gt;
```

Comment:
See https://forum.xwiki.org/t/html-codestyle-proposal-conventions-for-ids-and-classes/13619 . For CSS class names the kebab-case is preferred (and it's probably going to become a recommendation for IDs also).